contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
```

Comment:
minor: since we have a solid amount of links here, which should use the non-inline version of linking (`[foo][link]`), https://commonmark.org/help/tutorial/07-links.html